### PR TITLE
feature: add integration test for Informatieobjecten historie

### PR DESCRIPTION
**Pull request by hsiliev**
Checks the file history

Solves: PZ-3226

### CODE REVIEW

File: src/itest/kotlin/nl/lifely/zac/itest/InformatieobjectenHistorieTest.kt [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                    "applicatie": "ZAC",
+                    "attribuutLabel": "ondertekening",
+                    "door": "$TEST_USER_1_NAME",
+                    "toelichting": "Door ondertekenen"
+                  },
+                  {
+                    "applicatie": "ZAC",
+                    "attribuutLabel": "titel",
+                    "door": "$TEST_USER_1_NAME",
+                    "nieuweWaarde": "updated title with Špëcîål characters",
```
Comment (edgarvonk) [inline src/itest/kotlin/nl/lifely/zac/itest/InformatieobjectenHistorieTest.kt:78]: Could be a constant I guess?

**Review by edgarvonk — CHANGES_REQUESTED** [review]
nice one!

File: src/itest/kotlin/nl/lifely/zac/itest/InformatieobjectenHistorieTest.kt [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_PDF_FILE_NAME
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_SPEC_ORDER_AFTER_ZAAK_UPDATED
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_TXT_FILE_NAME
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_USER_1_NAME
+import nl.lifely.zac.itest.config.ItestConfiguration.TEXT_MIME_TYPE
+import nl.lifely.zac.itest.config.ItestConfiguration.ZAC_API_URI
+import nl.lifely.zac.itest.config.ItestConfiguration.enkelvoudigInformatieObjectUUID
+import nl.lifely.zac.itest.util.shouldEqualJsonIgnoringExtraneousFields
+
+@OptIn(ExperimentalKotest::class)
```
Comment (edgarvonk) [inline src/itest/kotlin/nl/lifely/zac/itest/InformatieobjectenHistorieTest.kt:34]: What is this and why do we need it?

Comment (hsiliev) [reply src/itest/kotlin/nl/lifely/zac/itest/InformatieobjectenHistorieTest.kt:34]: Needed for `shouldMatchSchema jsonSchema`. Kotest expects to change a bit or two (obj might become jsonObject and so on). Yet it should become productive in the end

Without jsonSchema part we have to extract objects and make checks more strict, which is problematic for dates and IDs

**Review by edgarvonk — APPROVED** [review]

**Pull request merged**